src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -539,20 +529,20 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
 
   override def setStatus(workflowId: WorkflowId, scopeKeys: Traversable[ExecutionDatabaseKey],
                          callStatus: CallStatus): Future[Unit] = {
+    val endTime = if (callStatus.isTerminal) Option(new Timestamp(Calendar.getInstance().getTime.getTime)) else None
```

Comment:
Pure whimsy on my part.  I prefer the conciseness of the Slick-provided implicit, I'll standardize on that.